dev/ColorPicker/APITests/ColorPickerTests.cs

```diff
@@ -232,6 +246,37 @@ public void ValidateHueRange()
             IdleSynchronizer.Wait();
         }
 
+        [TestMethod]
+        public void ValidateFractionalWidthDoesNotCrash()
+        {
+            ManualResetEvent spectrumLoadedEvent = new ManualResetEvent(false);
+
+            RunOnUIThread.Execute(() =>
+            {
+                ColorSpectrum colorSpectrum = new ColorSpectrum();
+                colorSpectrum.Width = 300.75;
+                colorSpectrum.Height = 300.75;
+                colorSpectrum.Loaded += (sender, args) =>
```

Comment:
This seems like a generally useful technique. Consider pulling it into a helper function for this test class.